### PR TITLE
Allow php 8

### PR DESCRIPTION
**Pull request by bronek89**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "type": "library",
     "license": "MIT",
     "require": {
-        "php": "^7.3"
+        "php": ">=7.4"
```
Comment (dlakomski) [inline composer.json:8]: I would rather set it to `^7.4 || 8.0.*`

**Review by dlakomski — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (dlakomski) [inline composer.json]: Why not `>= 7.3`?

Comment (bronek89) [reply composer.json]: no time to support 3 versions of php :P

**Review by dazet — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (dlakomski) [reply composer.json]: So 7.3 version should be removed from travis config as well.

**Pull request merged**